.vsts-pipelines/dotnet-buildtools-prereqs.yml

```diff
@@ -2,35 +2,9 @@ trigger: none
 pr: none
```

Comment:
What is the expected use case for this build pipeline going forward?

Reply:
After thinking about it, it may not really be necessary.  The only benefit it would have is if we needed to manually run a build and have it target a more specific path than one of the other pipelines.  I could go either way on this.

Reply:
My vote would be to proactively clean it up.  If it turns out that there is a use case for it, it would be simple to add back.